Tests/CreateAPITests/AllPackages/Package.swift

```diff
@@ -17,7 +17,6 @@ let package = Package(
     dependencies: [
         .package(path: "../Expected/cookpad"),
         .package(path: "../Expected/discriminator"),
-        .package(path: "../Expected/edgecases-change-access-control"),
```

Comment:
Think you also need to include the new package, unless you go back to keeping the same name